CreativeTechTaxonomy.json

```diff
@@ -430,23 +435,45 @@
           },
           "OBS" : {},
           "Wirecast" : {},
-          "vMix" : {}
+          "vMix" : {},
+          "Newtek NDI" : {}
         }
       }
     },
     "Lights" : {
       "Lighting Consoles" : {
-        "Object" : {}
```

Comment:
It seems like the json file is out of sync with the images in a couple places? For example
- the image shows Lighting Consoles > GrandMa
- the json has Lighting Consoles > Object

🤷🏼 

<img width="800" alt="Screen Shot 2022-01-05 at 9 34 17 PM" src="https://user-images.githubusercontent.com/2334552/148319083-0ae1de89-0d0b-4206-baf0-9fd263265af3.png">

Reply:
Yes it's true! The graph was actually all manually made in omnigraffle to begin with and this JSON was me just transcribing it over. Not an ideal process, but once it's transferred and formatted better, ideally future graphs can be just generated instead of manually updated